first_task_search_engine/prepare_dataset.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import pandas as pd
+from polyglot.detect import Detector
+
+
+def get_lang_info(text: str):
+    """
+    File contains questions in different languages.
+    We get only English texts.
+    """
+    try:
+        lang = Detector(text, quiet=True).language
+    except Exception:
```

Comment:
```suggestion
    except UnknownLanguage:
```
General Exceptions are not the best variant

Reply:
Get this in a new commit with changes: here can be two types of exception (unicode detection error and language detection error)

Reply:
commit: https://github.com/RisaMagpie/InformationalRetrievalPractice/commit/4ca95fd4ec90bcb0f9e21dfbd2248e92dfa33672